KVM/qemu/vfio_net_boot.cfg

```diff
@@ -0,0 +1,65 @@
+- vfio_net_boot:
+    virt_test_type = qemu
+    type = vfio_net_boot
+    start_vm = no
+    nics = ""
+    # Special host pci slots to be configured
+    # setup_hostdev_slots = 0000:00:00.1
+    hostdev_bind_driver = vfio-pci
+    vm_hostdev_driver = vfio-pci
+    ext_host = linux-ftp.intel.com
+    variants:
```

Comment:
Suggest add this variants "vm" "tdvm" in the last. so that the case name generates as "vfio_net_boot.tdvm.xxx".

Reply:
Thanks for reviewing, updated